source/blog/_posts/2017-09-25-ajax-panel.md

```diff
@@ -0,0 +1,92 @@
+---
+title: Ajax Panel
+tags:
+- frontend
+- ajax
+- javascript
+- jquery
+- framework
+
+categories:
+- dev
+
+authors: [lh]
+github_link: blog/_posts/2017-09-25-ajax-panel.md
+
+---
+
+In this blog post I want to present the concept of our ajax based panel system which we use in our B2B-Suite.
+In the B2B-Suite we had to develop a backend user interface inside the shopware frontend. 
+We evaluated different frameworks to achieve this target. AngularJS and Vue.JS were possible frameworks which allows
+two way data binding and statefull frontend access.
+
+Our target was also to use most of the already existing dependencies instead of adding new frameworks just for
+the B2B-Suite. As you maybe already know jQuery is a base dependency of Shopware 5.
+
+We decided to develop our own lightweight frontend framework which allows asynchronous HTTP requests for our frontend.
+The main target of this framework is to execute the asynchronous call and render the response in a given DOM element.
+
+This approach allows us to build front pages with one dom element. An additional feature is that we can still work
+under the http cache and be able to ignore the shopware cache ids.
+
```

Comment:
maybe you can use the image here https://docs.enterprise.shopware.com/b2b-suite/technical/ajax-panel/ to make it clearer what it's supposed to do?